encoding/encodeInto.any.js

```diff
@@ -1,3 +1,13 @@
+function createBuffer(type, length = 0) {
+  if (type === "ArrayBuffer") {
+    return new ArrayBuffer(length);
+  } else {
```

Comment:
Since this is a test, IMO it's worth doing an explicit test that type === "SharedArrayBuffer", and throwing an exception if it's neither of the two. (A switch/case would also work.)